src/components/ha-markdown.ts

```diff
@@ -50,9 +50,14 @@ class HaMarkdown extends UpdatingElement {
       // Open external links in a new window
       if (
         node.nodeName === "A" &&
-        (node as HTMLAnchorElement).host !== document.location.host
+        node instanceof HTMLAnchorElement &&
```

Comment:
This check seems not necessary, as we already checked that the nodeName is `A` ?